### PR TITLE
Adds changelog entries for v3.13.0

### PR DESCRIPTION
**Pull request by joehan**
### Description
Adds CHANGELOG entries for v3.13.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+- Adds `serviceAccount` option to `runtimeOptions`, which lets users to specify what service account a Cloud Function should run as. For example: ```
```
Comment (samtstern) [inline CHANGELOG.md:1]:
```suggestion
- Adds `serviceAccount` option to `runtimeOptions` to specify which service account Cloud Function should use at runtime. For example: ```
```

Comment (samtstern) [inline CHANGELOG.md:1]: I don't think inline ``` is legal

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  const functions = require('firebase-functions');
+
+functions.runWith({
+serviceAccount: 'test-sa@project.iam.gserviceaccount.com'
+// OR
+// serviceAcount: 'test-sa@"
+// OR
+// serviceAccount: 'default'
+})
```
Comment (samtstern) [inline CHANGELOG.md:10]:
```suggestion
exports.myFunction = functions.runWith({
    serviceAccount: 'test-sa@project.iam.gserviceaccount.com'
    // OR
    // serviceAcount: 'test-sa@"
    // OR
    // serviceAccount: 'default'
  })
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+```
+Requires firebase-tools@8.18.0 or later. Thanks @egor-miasnikov!
+- Upgrades `highlight.js` to `10.4.1` to fix a vulnerability.
+```
```
Comment (samtstern) [inline CHANGELOG.md:15]:
```suggestion
```

**Review by samtstern — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (joehan) [reply CHANGELOG.md]: Moved this to its own line, I think this should work now.

**Pull request merged**